src/applications/vre/28-8832/components/GetFormHelp.jsx

```diff
@@ -7,11 +7,11 @@ import Telephone, {
 const GetFormHelp = () => (
   <p className="help-talk">
     If you have questions or need help filling out this form, please call our
-    MYVA411 main information line at <Telephone contact={CONTACTS.VA_311} /> and{' '}
-    select 0. We’re here 24/7.
-    <br />
+    <span aria-label="my VA 4 1 1.">MYVA411</span> main information line at{' '}
+    <Telephone contact={CONTACTS.VA_311} /> and select 0. We’re here{' '}
```

Comment:
I don't think these are the same numbers... sorry for the confusion!

Reply:
🤷  Not sure. @peggygannon provided us with the number 800-698-2411, and it seems to line up with the number here - https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation-react/src/components/Telephone/contacts.js#L81

Reply:
Weird that the label includes 311, but the number is 411...